Added loggers. closes #20

sk364 opened the pull request:
3 commands need to be run for this with sudo access in setup phase:
```
mkdir /var/log/hs-api
touch /var/log/hs-api/debug.log
chown -R <user>:<user> /var/log/hs-api
```

sk364 committed:
Added loggers. closes #20

Samy-33 commented on api/settings/dev.py:
```diff
@@ -1,5 +1,15 @@
-ALLOWED_HOSTS = ['*']
\ No newline at end of file
+# hack to disable autoreload logger
+l = logging.getLogger('django.utils.autoreload')
```
Just use: `logging.getLogger('django.utils.autoreload').disabled = True`  
unless we are using `l` for `None` check

Samy-33 commented on api/settings/prod.py:
```diff
@@ -176,6 +177,64 @@
+        'debug_file': {
+            'level': 'DEBUG',
+            'class': 'logging.FileHandler',
+            'filename': '/var/log/hs-api/debug.log',
```
Create a variable `LOG_FILE_PATH`
* It breaks because there is no `/var/log/hs-api/` directory. We need to create the directory while project setup. 
* Is there a way to create the file automatically? we can have this file in `~/.logs/hs-api/` to tackle root permission issues.
```
FileNotFoundError: [Errno 2] No such file or directory: '/var/log/hs-api/debug.log'
```

sk364 replied:
In dev/prod, the created user (non-root) will have sudo access. So, I don't think there should be any concern about root permissions.

3 commands need to be run for this with sudo access in setup phase:
```
mkdir /var/log/hs-api
touch /var/log/hs-api/debug.log
chown -R <user>:<user> /var/log/hs-api
```

Samy-33 reviewed (changes requested):
Also, please resolve the conflicts.

sk364 committed:
Refactored code
- Usage of shortcut function `getenv`, instead of `environ.get`
- Updated log handler `django.db.backends` to propagate only when DEBUG is True
- Added global variable for debug file name
- Removed hack: Updated `django` handler's log level to INFO

sk364 committed:
Resolved conflicts
